client-course-schedulizer/src/components/Toolbar/ScheduleToolbar/ScheduleToolbar.scss

```diff
@@ -14,8 +14,8 @@
   padding: $spacing;
   margin-left: -2 * $spacing;
   margin-right: -2 * $spacing;
-  // TODO: there is slight squishing when stickied but not sure how to fix. takes time to debug
-  margin-bottom: 1.5 * $spacing;
+  // TODO: there is slight squishing when scrolling down before sticky headers get picked up
+  margin-bottom: 2.5 * $spacing;
```

Comment:
this makes the header jerk on my mac. I know you were experiencing issues will the search bar being cut off, so maybe you could look into increasing the z-index on the search bar.